frontend/public/components/utils/units.js

```diff
@@ -122,10 +122,9 @@ const getDefaultFractionDigits = value => {
   return 1;
 };
 
-const round = units.round = (value, options) => {
-  const fractionDigits = getDefaultFractionDigits(value);
+const formatValue = (value, options) => {
   const {locales, ...rest} = _.defaults(options, {
-    maximumFractionDigits: fractionDigits,
+    maximumFractionDigits: 20,
```

Comment:
What is the reason for this change? 20 seems like a big value, but maybe I'm misunderstanding this option.

Reply:
fraction digits are determined by `round` function https://github.com/openshift/console/blob/21c5ba2b87b0fbf97e4f7034e27b5dd2fe0be5d6/frontend/public/components/utils/units.js#L140 and `formatValue` function is used only for formatting string part - I didnt want to change value here so I set `maximumFractionDigits` to maximum that is allowed

Reply:
Do we need `round` at all if we can use `maximumFractionDigits` for formatting here?

This should at least have a comment explaining.

Reply:
humanize functions return object
```
value: number;
unit: string;
string: string;
```
at first we work on `value` -> it gets rounded (result is number type) and then the rounded value is passed to `formatValue` which is using `Intl.NumberFormat.format` and the result is string type.

I we use only `Intl.NumberFormat.format` then we would need to parse the resulting string back to number which seems a more clumsy to me.

I will add a comment explaining this if we decide to go with current approach

Reply:
So I'm still unclear why we need to change this implementation here. Is this fixing a bug, changing behavior, or just refactoring? Can you give an example of the before / after with this change?

Based on your description, I think the existing API is a bit confusing. I would expect

* A function that takes a string value with units and returns an object like `{value, unit}`
* A function that takes the base, unitless value, picks the best unit, and formats it using `Intl.NumberFormat` and specifying significant digits
* A function that takes `{value, unit}` and formats it using `Intl.NumberFormat` and specifying significant digits

I'm not proposing we make these changes in this PR, but we might look at refactoring this down the road.

Reply:
If we keep the `fractionDigits` in `Intl.NumberFormat` config and call `humanize` with `useRound=false` we will end up with `value` not rounded, but `string` rounded. Maybe that is not an issue, `useRound=false` isnt even used anywhere in code right now and when we refactor things in the future we will either way end up with something different.

Reply:
> If we keep the fractionDigits in Intl.NumberFormat config and call humanize with useRound=false we will end up with value not rounded, but string rounded. 

This is exactly what I would expect :) I think it's a better API. Is this what the code was doing before?

Reply:
Ok :) Reverted the change. Before we started using `Intl.NumberFormat` both `value` and `string` was the same (`string` was just `value` + `space` + `unit`), after `Intl.NumberFormat` was introduced the things got different/tests started to fail so I wasnt sure if this change was on purpose or not. But I agree that it makes sense how to keep the logic as is.

Reply:
OK, I see the complaint now. It's the the string value is rounded when useRound = false. Sorry I misunderstood.

I think the fix is to always round the humanized value and just remove `useRound` if it's not used anywhere today. But that can be a follow on.